build.sbt

```diff
@@ -15,33 +14,34 @@ def scalaBinaryVersion(scalaVersion: String) = scalaVersion match {
   case version if version startsWith "2.12" => "2.12"
   case _ => sys.error("unknown error")
 }
-def isScala210x(scalaVersion: String) = scalaBinaryVersion(scalaVersion) == "2.10"
 
 val algebirdVersion = "0.13.0"
 val apacheCommonsVersion = "2.2"
 val avroVersion = "1.7.4"
-val bijectionVersion = "0.9.1"
+val bijectionVersion = "0.9.5"
 val cascadingAvroVersion = "2.1.2"
-val chillVersion = "0.8.3"
+val chillVersion = "0.8.4"
```

Comment:
@johnynek I can switch this to 0.7.7. If I understand correctly the plan would be:
1) Merge this PR with chill 0.7.7 (might need to revert the test fix here: https://github.com/twitter/scalding/pull/1634/files).
2) Cut a 0.17.0 release 
3) You can use that to cut a SB 2.12 rel on chill 0.7.x
4) Switch back to chill 0.8.4
5) Cut a 0.17.1 rel. 
6) We can use this for our SB 2.12 rel on chill 0.8.x

@ttim - does this sound reasonable to you as well?